phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/installations/dto/InstallationDTO.java

```diff
@@ -61,16 +65,19 @@ private InstallationDTO(Builder builder) {
         node = builder.node;
     }
 
+    /**
+     * type of the installation
+     */
     public enum InstallationType {
-        APPS, WINE;
+        APPS, ENGINES;
 
         @Override
         public String toString() {
             switch (this) {
```

Comment:
It's shorter and easier extendable. Let's say you want to add a new enum type later on, then it's possible you forget to add the new case to the `toString` method. If you modify the enum to take a display string with a constructor this can't happen.